smtp/smtpc.c

```diff
@@ -440,10 +450,25 @@ smtpc_execute(struct smtpc_request *req, double timeout)
 		++env->stat.failed_requests;
 		smtpc_request_delete(req);
 		return -1;
+	case CURLE_SEND_ERROR:
+	case CURLE_RECV_ERROR:
+		curl_easy_getinfo(req->easy, CURLINFO_RESPONSE_CODE, &longval);
+		req->status = (int) longval;
+
+		/** When client can not read bytes from another side that
+		 * leads to a message about failed response reading but
+		 * response code stays 250, that is not correct.
+		 */
```

Comment:
This is not a documentation comment, but rather a code comment (don't know how to name it properly, but I guess you spotted the difference). So it should start from `/*`.

Reply:
Fixed.